room/compiler/src/test/kotlin/androidx/room/processor/QueryMethodProcessorTest.kt

```diff
@@ -946,12 +969,7 @@ class QueryMethodProcessorTest(val enableVerification: Boolean) {
         ) { _, _, _ ->
         }?.failsToCompile()
             ?.withErrorContaining("no such column: age")
-            ?.and()?.withErrorContaining(
-                cannotFindQueryResultAdapter(
-                    ClassName.get("foo.bar", "MyClass", "Pojo")
-                )
-            )
-            ?.and()?.withErrorCount(2)
+            ?.and()?.withErrorCount(1)
```

Comment:
I don't think this needs to be removed, when db verification is 'off' the `pojoTest` does not return the 'assertion subject' which means this 'withError' assertions are only when verification is ON, for which we want Room to fail since with column verification the Pojo does not match the result.